### PR TITLE
New domains added

### PR DESCRIPTION
**Pull request by NGTmeaty**
i think i did the gulp stuff right

### CODE REVIEW

File: public-cdn-domains.txt [hunk 1]
```diff
@@ -6,8 +6,12 @@ files:*.is-fi.re
 files:*.is-serious.business
 files:*.is-pretty.cool
 files:*.is-pretty.sexy
+files:*.is-bad.com
```
Comment (deansheather) [inline public-cdn-domains.txt]: Domains in this file should be in alphabetical order.

File: public-cdn-domains.txt [hunk 2]
```diff
@@ -6,8 +6,12 @@ files:*.is-fi.re
...
+files:*.is-a-bad-waifu.com
+files:*.is-a-good-waifu.com
 
 files:owo.whats-th.is
+files:owo.solutions
```
Comment (deansheather) [inline public-cdn-domains.txt]: `owo.whats-th.is` and `awau.moe` circumvent the alphabetical ordering thing, move this down to be where it should be in relation to the other `files:` domains.

File: src/views/faq.pug [hunk 3]
```diff
@@ -96,6 +96,7 @@ block content
     h4 You can find the file hosting service on:
     ul
       li #[code owo.whats-th.is]
+      li #[code owo.solutions]
```
Comment (deansheather) [inline src/views/faq.pug]: Should be in alphabetical order.

**Comment by deansheather** [comment]
Also, just clone the repository and use git to push. No need to upload files manually.

**Comment by NGTmeaty** [comment]
has new domains from Wolke

**Comment by deansheather** [comment]
You still haven't fixed my requested changes. I'm going to close this and do it myself.

**Pull request closed without merging**